### PR TITLE
Imaging GUI: Merge the Run and Setup tabs

### LINKED ISSUE #18134: Imaging GUI: Merge Run and Setup tabs
Run tab has an additional image browsing which is unnecessary.

Setup tab functionality can be merged within the Run tab functionality.

This _requires_ merged first: 
- [x] #18095 
- [ ] #18258

### PR DESCRIPTION
**Pull request by dtasev**
- [x] Requres #18258 merged first

---

## Description of work.

- The tabs that were previously `Run` and `Setup` separately, have now been merged into a single one, just called `Run`.
   - Any suggestions to further improve/change the merge are welcome.
- The tab ordering of the tab stops in the `Run` tab has been improved.

## To test:

- Test functionality from both Run and Setup tabs 
  - Selecting a different tool and path works when running a reconstruction
- Check that the ordering when pressing Tab is sensible (it used to jump from password to reconstruction tool to username, etc., which is bad for navigation)

Fixes #18134 .

[*Added in imaging.rst file*: 5180d1](https://github.com/mantidproject/mantid/commit/5180d1453c56a1f8717b3adaf9ef03a9dc547223)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by raquelalvarezbanos — COMMENTED** [review]
I haven't had much time to review code changes, but from what I've seen, changes look good. I'll test the functionality later.

File: MantidQt/CustomInterfaces/src/Tomography/TomographyROIViewQtWidget.cpp [hunk 1]
```diff
@@ -709,33 +704,38 @@ void ImageROIViewQtWidget::updatePlay() {
...
+void TomographyROIViewQtWidget::rotationUpdated(int /* idx */) {
   m_params.rotation =
       static_cast<float>(m_ui.comboBox_rotation->currentIndex()) * 90.0f;
-  m_presenter->notify(ImageROIPresenter::ChangeRotation);
+  m_presenter->notify(TomographyROIPresenter::ChangeRotation);
+}
+
+void TomographyROIViewQtWidget::autoCoRToolChanged(int /* idx */) {
+  // todo notify presenter that the tool is changed, however for now we only
+  // have one tool -> tomopy
```
Comment (DimitarTasev) [inline MantidQt/CustomInterfaces/src/Tomography/TomographyROIViewQtWidget.cpp]: Yes this is connected to changing the tool method, and will be used when(if) there is more than one available.

File: MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Tomography/ITomographyROIPresenter.h [hunk 2]
```diff
@@ -58,7 +58,6 @@ class ITomographyROIPresenter {
     ColorRangeUpdated,     ///< The uses changes the values or options of the
                            ///< color range
     SelectCoR,             ///< Start picking of the center of rotation
-    FindCoR,               ///< Run calculate CoR
```
Comment (raquelalvarezbanos) [inline MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Tomography/ITomographyROIPresenter.h]: What happened to this and changes below? were they not meant to be part of this PR?

Comment (DimitarTasev) [inline MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Tomography/ITomographyROIPresenter.h]: These changes were from #18258. I've removed them in that PR as well as they were unused code. The only changes this PR covers is the merge of the tabs

**Comment by raquelalvarezbanos** [comment]
Looks good: "Run" and "Setup" were merged into "Run" tab. I think the new display looks good (options from the old "setup" are now displayed at the beginning, and below is the list of reconstructed jobs). 
When pressing Tab the ordering seems reasonable and the next widget in the group is selected. I was able to change to different reconstruction tools and run the reconstruction locally. Code changes look also good, some come from PR https://github.com/mantidproject/mantid/pull/18258, which needs to be merged first. I'll wait until then before adding the squirrel.

**Comment by raquelalvarezbanos** [comment]
Before I forget, I noticed that the documentation seems a bit out dated. Is it going to be updated in a future PR?

**Comment by DimitarTasev** [comment]
@raquelalvarezbanos Yes, there's a lot of changes in #18379 that will have to be documented as well, I will update all of the documentation in one sweep there!

**Comment by DimitarTasev** [comment]
@raquelalvarezbanos Could you mark with shipit, if there are no further comments?

**Comment by raquelalvarezbanos** [comment]
:shipit:

**Comment by martyngigg** [comment]
Remaining build failures are unrelated.

**Pull request merged**